python_modules/libraries/dagster-dbt/dagster_dbt/types.py

```diff
@@ -52,7 +52,7 @@ class NodeResult(
         warn (Optional[Any]): The ``warn`` field from the results of the executed dbt node.
         skip (Optional[Any]): The ``skip`` field from the results of the executed dbt node.
         status (Optional[Union[str,int]]): The status of the executed dbt node (model).
-        execution_time (float): The execution duration (in seconds) of the dbt node (model).
+        execution_time (Union[float,int]): The execution duration (in seconds) of the dbt node (model).
```

Comment:
Nit: this should actually always be float since we convert it before we construct the namedtuple.